_includes/sidenav.html

```diff
@@ -10,13 +10,12 @@
    Create nav markup when given:
 
    items:               An array of tuples used to display nav items.
-                        format: (href, id, caption)
+                        format: (href, id, caption, children)
 
    active_item_id:      A string indicating the id of the nav item
                         to be activated.
 
-   additional_classes:  Extra classes you wish to add to the nav,
-                        space separated.
+   additional_classes:  Extra classes you wish to add to the nav, space separated.
```

Comment:
Seems to have fixed itself in  809b220. Guessing I had it saved but not added when I commited.